README.txt

```diff
@@ -706,6 +706,260 @@
 
             python2.7 python/computeEffCurves.py --file=config/configEffPredictor.cfg --deubg=True
 
+
+	# 3.c.iii Analysis of QC data stored in an excel file
+        # --------------------------------------------------------
+	main script: Produce_Config_File.py
+	additional scripts: QC3.py, QC4_HV.py, QC4_SS.py, QC5_Gain.py, QC5_Rate.py
```

Comment:
Make a separate section in the README which describes these separate scripts.  Here you have only described `Produce_Config_File.py`